pallets/lockdown-mode/src/tests.rs

```diff
@@ -22,7 +38,7 @@ fn activate_lockdown_mode_works() {
 
 #[test]
 fn deactivate_lockdown_mode_works() {
-	new_test_ext().execute_with(|| {
+	new_test_ext(false).execute_with(|| {
 		// We activate lockdown mode first so we can deactivate it.
 		assert_ok!(LockdownMode::activate_lockdown_mode(RuntimeOrigin::root()));
```

Comment:
We can set initial state to `true` and remove this line

Reply:
Done.